PackageDescriptionConfigs/eos_package_description.json

```diff
@@ -79,26 +79,26 @@
         {"ignore_regex":"Assets/Plugins/Windows/.*/steam_api64\\.dll"},
 
         {"comment": "----------------------------End Windows---------------------------------------------"},
-
-        {"src":"Assets/Plugins/Linux.meta", "dest": "Runtime/"},
-        
-        {"src":"Assets/Plugins/Linux/license.txt", "dest": "Runtime/Linux/"},
+		
+		{"src":"Assets/Plugins/Linux.meta", "dest": "Runtime/"},
+		
```

Comment:
This might be a small thing, but I notice inconsistent tabs or spaces in this file now.